crates/bevy_picking/src/pointer.rs

```diff
@@ -29,41 +27,69 @@ use crate::backend::HitData;
 #[derive(Debug, Default, Clone, Copy, Eq, PartialEq, Hash, Component, Reflect)]
 #[require(PointerLocation, PointerPress, PointerInteraction)]
 #[reflect(Component, Default, Debug, Hash, PartialEq)]
-pub enum PointerId {
-    /// The mouse pointer.
-    #[default]
-    Mouse,
-    /// A touch input, usually numbered by window touch events from `winit`.
-    Touch(u64),
-    /// A custom, uniquely identified pointer. Useful for mocking inputs or implementing a software
-    /// controlled cursor.
-    #[reflect(ignore)]
-    Custom(Uuid),
+pub struct PointerId {
+    /// A stable identifier for a specific pointer.
+    ///
+    /// This is typically provided by winit,
+    /// and is unique for each touch input or mouse.
```

Comment:
The id here is for multitouch corresponding to a specific touch on the device and it's only guaranteed to be unique for the duration of that touch I think. It doesn't feel correct to compare a single touch to a mouse device?